plugins/@grouparoo/salesforce/src/migrations/000002-plugin-salesforce-export-name.ts

```diff
@@ -0,0 +1,20 @@
+import Sequelize from "sequelize";
```

Comment:
Not sure if it matters, but this one will go in soon too https://github.com/grouparoo/grouparoo/pull/2346 and it has `002`

Reply:
Updated!